### PR TITLE
Local file policy adaptation

### PR DESCRIPTION
**Pull request by joroca**
- Form section in jsonform object.
- group_by_multiple_keys method has been modified due to some policy form fields which may change according to user selection (add/remove).

### CODE REVIEW

File: gecoscc/tasks.py [hunk 1]
```diff
@@ -770,18 +770,14 @@ def group_by_multiple_keys(self, input_data, mergeIdField, mergeActionField, opp
...
-              temp_dict[mergeActionField] += [aux.pop(mergeActionField)]
-              temp_dict.update(aux)
-           if opposite:
-              # Incoherence in node
-              # temp_dict[mergeActionField] = ['add','remove']
-              if len(temp_dict[mergeActionField]) > 1:
-                  continue
-           temp_dict[mergeActionField] = temp_dict[mergeActionField][-1] # input_data sorted (ous,groups,computers). Last item has the highest priority
-           result.append(temp_dict)
+            group = list(grp)
```
Comment (alfem) [inline gecoscc/tasks.py]: Are these changes related to the PR?

Comment (joroca) [reply gecoscc/tasks.py]: From now on, policies will have a different number of fields depending on the action (add/remove) that user choose in policies forms.

Comment (alfem) [reply gecoscc/tasks.py]: That is an important change. We need to test this thoroughly
